src/modules/articles/components/ArticleBody.tsx

```diff
@@ -109,37 +108,71 @@ interface IProps {
   article: IArticle
 }
 
-const ArticleBody: React.FunctionComponent<IProps> = ({ classes, article }) => {
-  // TODO: refactor media to make the carousel part of media/back-end
-  const isCarouselButtonVisible =
-    SPEC_IMG_CAROUSEL_PATTERN.test(article.content) && article.media && article.media.length > 0;
+class ArticleBody extends React.Component<IProps> {
+  constructor(props: IProps) {
+    super(props);
 
-  return (
-    <Row>
-      <Col xs={12} sm={12} md={8} lg={8} className={classes.ArticleBody}>
-        {SPEC_IMG_CAROUSEL_PATTERN.test(article.content) && (
-          <Lightbox title={article.title}>
-            <Gallery media={article.media} />
-          </Lightbox>
-        )}
-        {article.media && article.media.length > 0 && (
-          <ArticleFeaturedMedia
-            image={article.media[0]}
-            isCarouselButtonVisible={isCarouselButtonVisible}
-            carouselImageCount={article.media.length}
-          />
-        )}
-        <ArticleReference article={article} />
-        <div
-          className={classes.innerHTML}
-          dangerouslySetInnerHTML={{ __html: article.content }}
-        />
-      </Col>
-      <Col xsHidden smHidden mdOffset={1} md={3} lgOffset={1} lg={3}>
-        <RightRail />
-      </Col>
-    </Row>
-  );
-};
+    //creates div in which to store article contents
+    this.articleContentDiv = document.createElement("div");
+
+    this.createArticleContent = this.createArticleContent.bind(this);
+  }
+
+  private createArticleContent(element: HTMLDivElement | null): any {
```

Comment:
why `any`?

Reply:
That was part of the type that react expects in `ref={...}`, but you're right I can omit it since I return void.